src/IntlPhoneInput.js

```diff
@@ -119,6 +119,11 @@ export default class IntlPhoneInput extends React.Component {
     this.showModal();
   }
 
+  componentDidMount() {
+    this.props.inputRef.current.focus();
```

Comment:
To be removed, it should be userland's role to do this

Reply:
Done